x-pack/plugins/monitoring_collection/server/routes/api/v1/dynamic_route/index.test.ts

```diff
@@ -42,7 +42,7 @@ describe('dynamic route', () => {
     const router = httpServiceMock.createRouter();
 
     const getMetric = async () => {
-      return { foo: 1 };
+      return Promise.resolve({ foo: 1 });
```

Comment:
Why the change here? The tests seem to pass with regular objects. Since it's an async function I wouldn't expect you'd need to return a promise.

Reply:
Nevermind, I reverted it.